msg/messages.js

```diff
@@ -766,6 +766,30 @@ Blockly.Msg.TEXT_PROMPT_TOOLTIP_NUMBER = 'Prompt for user for a number.';
 /// https://github.com/google/blockly/wiki/Text#printing-text].
 Blockly.Msg.TEXT_PROMPT_TOOLTIP_TEXT = 'Prompt for user for some text.';
 
+/// message0 interpolation string
+Blockly.Msg.TEXT_COUNT_MESSAGE0 = 'count %1 in %2';
+/// url - Information about counting how many times a string appears in another string.
+Blockly.Msg.TEXT_COUNT_HELPURL = '';
+/// tooltip - See [https://github.com/google/blockly/wiki/Text#printing-text
+/// https://github.com/google/blockly/wiki/Text#printing-text].
+Blockly.Msg.TEXT_COUNT_TOOLTIP = 'Count how many times a string occurs in another string.';
+
+/// message0 interpolation string
+Blockly.Msg.TEXT_REPLACE_MESSAGE0 = 'replace %1 with %2 in %3';
+/// url - Information about replacing a string within another string.
+Blockly.Msg.TEXT_REPLACE_HELPURL = '';
+/// tooltip - See [https://github.com/google/blockly/wiki/Text#printing-text
+/// https://github.com/google/blockly/wiki/Text#printing-text].
+Blockly.Msg.TEXT_REPLACE_TOOLTIP = 'Replace a string within another string.';
```

Comment:
As a user I'd like to know if this replaces every instance or only the first instance.

Also, Blockly uses the word 'text' rather than 'string'.  Sometimes that sounds weird to us, but the average Blockly user has no idea what a string is.

And the comments in this file are important, they are the descriptions that the translators use.  So make sure they are accurate and make sense.  See others for examples.